### PR TITLE
[BUGFIX] Small bugs in 'enumerate_cycle_basis' and relative_setup 'run'

### PR DESCRIPTION
**Pull request by brycestx**
Correcting small bugs in '[enumerate_cycle_basis](https://github.com/choderalab/perses/blob/master/perses/rjmc/topology_proposal.py#L1578)' function in topology_proposal.py and '[run](https://github.com/choderalab/perses/blob/master/perses/dispersed/relative_setup.py#L468)' function in relative_setup.py that I found while trying to debug #448 

The correction in the run function was referenced here #445 but I think it may have never been added to master.

### CODE REVIEW

File: perses/rjmc/topology_proposal.py [hunk 1]
```diff
@@ -1604,7 +1604,7 @@ def enumerate_cycle_basis(molecule):
                 for i in range(len(cycle)):
                     atom_index_1 = cycle[i]
                     atom_index_2 = cycle[(i+1)%len(cycle)]
-                    edge = g.edges[atom_index_1,atom_index_2]
+                    edge = g.edge[atom_index_1][atom_index_2]
```
Comment (jchodera) [inline perses/rjmc/topology_proposal.py]: @brycestx : Are you using `networkx` 1.x or 2.x? I thought the API changed for 2.0, and that's what I was using here. Or are both `Graph.edge` and `Graph.edges` available in 2.0?

Comment (brycestx) [reply perses/rjmc/topology_proposal.py]: It does appear that my networkx is 1.11. I just upgraded and it works now. My bad, we should add a requirement in setup.py to address this

Comment (jchodera) [reply perses/rjmc/topology_proposal.py]: We've had some trouble with requirements in `setup.py` that can't be installed via `pip`---I wonder if there's a way to just add a function to `setup.py` that checks to make sure all the requirements in `devtools/conda/meta.yaml` are installed and suggest you install them via `conda` if they aren't present.

**Comment by brycestx** [comment]
Gonna close this due to fix of updating networkx. I'll just file an issue for the #445 referenced bug due to the fact that it looks like dask throws an error due to it. However I've been having a bunch of issues with dask, where my workers just get killed.. so I will try to put together an issue for that

**Comment by jchodera** [comment]
Great! Please file a separate issue for each problem/bug/question if possible---it makes it easier to track progress toward resolution that way!

**Pull request closed without merging**